frontend/tests/unit/routes/scrollable-route-test.js

```diff
@@ -0,0 +1,11 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Route | scrollableRoute', function(hooks) {
+  setupTest(hooks);
+
+  test('it exists', function(assert) {
+    let route = this.owner.lookup('route:scrollable-route');
+    assert.ok(route);
+  });
+});
```

Comment:
I'd acceptance test this instead, delete this test (it doesn't do anything)

Reply:
Going to acceptance test this when Godfrey has merged the acceptance test setup for transportation